cypress/plugins/index.js

```diff
@@ -1,17 +1,13 @@
-// ***********************************************************
-// This example plugins/index.js can be used to load plugins
-//
-// You can change the location of this file or turn off loading
-// the plugins file with the 'pluginsFile' configuration option.
-//
-// You can read more here:
-// https://on.cypress.io/plugins-guide
-// ***********************************************************
+const getChallenge = require('../../curriculum/getChallenges');
 
-// This function is called when a project is opened or re-opened (e.g. due to
-// the project's config changing)
-/* eslint-disable no-unused-vars */
-module.exports = (on, config) => {
-  // `on` is used to hook into various events Cypress emits
-  // `config` is the resolved Cypress config
+module.exports = on => {
+  on('task', {
+    // return an array of all challenge paths
+    //  superBlock, block,  this are unused variables in the condition Husky doesn't like that
+    getAllChallengePaths({ lang }) {
+      const curriculum = getChallenge.getChallengesForLang(lang);
+
+      return curriculum['responsive-web-design'];
```

Comment:
```suggestion
      return getChallenge.getChallengesForLang(lang);
```
Since `getChallengesForLang` returns a Promise, the easiest thing to do is just return that here.  `on` can cope with tasks that return Promises, so we can let it handle this.